mqtt/docker/linux/amd64/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: dockerfile -*-
+#
+# An example Dockerfile showing how to build a Rust executable using this
+# image, and deploy it with a tiny Alpine Linux container.
+
+# You can override this `--build-arg BASE_IMAGE=...` to use different
+# version of Rust or OpenSSL.
+ARG BASE_IMAGE=ekidd/rust-musl-builder:stable
+
+# Our first FROM statement declares the build environment.
+FROM ${BASE_IMAGE} AS builder
+
+# Add our source code.
+ADD . ./
+
+# Fix permissions on source code.
+# RUN sudo chown -R rust:rust /home/rust
+
+# Build our application.
+RUN cargo build -p mqttd --release && strip /home/rust/src/target/x86_64-unknown-linux-musl/release/mqttd
+
+FROM scratch
+ENV RUST_LOG=info
+EXPOSE 1883/tcp
```

Comment:
why need this?

Reply:
@philipktlin  do you mean `EXPOSE`? it is just a convention to let consumers know that the container exposes these ports. Do you have any concerns about that?

Reply:
Do we need to use 1883 port? For mqtt in edgeHub, we use port 8883 only.

Reply:
Yes we need, it is MQTT no TLS port, which we will support.

Reply:
check last sentence in overview section in https://en.wikipedia.org/wiki/MQTT.  8883 is used for MQTT over SSL (using TLS) and 1883 is MQTT over TCP.  And edgeHub is only listening to 8883.  Should MQTT broker also listen to 8883 only?

Reply:
Just want to confirm that when release mqtt-broker, port 1883 should not be needed, right? why do we want to run perf and integration test using 1883? @dmolokanov could you clarify?

Reply:
Actually it will be used for some clients. Some devices are not powerful enough to support any encryption. It'll special case, but we want to support them.

Reply:
Do you mean mqtt broker will support both ports, but IoT Edge will support 8883 only?  And would you please give some example of clients will use port 1883?